scipy/stats/stats.py

```diff
@@ -5452,8 +5453,10 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
     weights, which can make Stouffer's method more powerful than Fisher's
     method when the p-values are from studies of different size [3]_ [4]_.
     The Pearson's method uses :math:`log(1-p_i)` inside the sum whereas Fisher's
-    method uses :math:`log(p_i)` [6]_. The `mudholkar_george` method is the
-    difference of the Fisher's and Pearson's test statistics.
+    method uses :math:`log(p_i)` [6]_. Fisher's and Pearson's methods each
```

Comment:
Maybe clarify: "For Fisher's and Pearson's method, the sum of the logarithms is multiplied by -2 in the implementation. This quantity has a `chisquare` distribution that determines the p-value."

Reply:
you got it.